map.go

```diff
@@ -155,6 +155,15 @@ type MapKV struct {
 //
 // Returns an error wrapping [ErrMapIncompatible] otherwise.
 func (ms *MapSpec) Compatible(m *Map) error {
+	maxEntries := ms.MaxEntries
+	if ms.Type == PerfEventArray {
+		var err error
+		maxEntries, err = fixupPerfEventArraySize(maxEntries)
+		if err != nil {
+			return err
```

Comment:
Should we wrap this? Otherwise, the caller will receive the raw error from internal.PossibleCPUs without any context.

Reply:
Wrapped in `fixupPerfEventArraySize` instead.